model-optimizer/extensions/front/tf/TFFFTToDFT.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+from extensions.ops.dft import DFT, IDFT
+from mo.front.common.partial_infer.utils import int64_array
+from mo.front.common.replacement import FrontReplacementSubgraph
+from mo.front.tf.graph_utils import create_op_with_const_inputs
+from mo.graph.graph import Graph, rename_nodes
+
+
+class TFFFTToDFT(FrontReplacementSubgraph):
+    """
+    This transformation converts the operation TFFFT into MO DFT (if the attribute 'is_inverse' is False),
```

Comment:
nit: actually, I would call it `OpenVINO DFT` instead of `MO DFT`

Reply:
Fixed.